pkg/standalone-utils/contracts/relayer/BaseRelayerLibrary.sol

```diff
@@ -81,6 +81,15 @@ contract BaseRelayerLibrary is IBaseRelayerLibrary {
         token.approve(address(getVault()), amount);
     }
 
+    /**
+     * @notice Returns the amount referenced by chained reference `ref`.
+     * @dev It does not alter the reference (even if it's marked as temporary), and it does not expose the reference
+     * slot (see `_peekChainedReferenceValue`).
```

Comment:
```suggestion
     * @dev It does not clear the reference (even if it's marked as temporary).
```

Saying 'it doesn't do X' seems a bit out of scope - you wouldn't expect for this function to return the slot.